client/src/components/fields/select-pages.js

```diff
@@ -149,16 +172,20 @@ export class SelectPages extends React.Component {
 		}
 
 		return (
-			<Select
+			<Creatable
 				name={this.props.name}
 				disabled={disabled}
 				value={this.state.value}
 				multi={true}
 				isLoading={loading}
 				options={this.state.options}
-				onInputChange={this.onInputChange.bind( this )}
-				onChange={this.onChange.bind( this )}
-				filterOptions={this.filterOptions.bind( this )}
+				onInputChange={this.onInputChange}
+				onChange={this.onChange}
+				filterOption={option => option}
```

Comment:
I think this is the default behavior if I'm not mistaken.

Reply:
it looks like the filtering is _always_ enabled unless you disable it by a function, but I just realized the function should return a bool, so updating. https://github.com/JedWatson/react-select#filtering-options